app/sidekiq/evss/disability_compensation_form/form526_document_upload_failure_email.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'va_notify/service'
+
+module EVSS
+  module DisabilityCompensationForm
+    class Form526DocumentUploadFailureEmail < Job
+      STATSD_SENT_METRIC_KEY = 'api.form_526.document_upload_failure_notification_sent'
+      STATSD_EXHAUSTED_METRIC_KEY = 'api.form_526.document_upload_failure_email_job_exhausted'
```

Comment:
I believe there may already be a pattern in place that appends an outcome (e.g., `notification_sent`) to a `STATSD_KEY_PREFIX` key when `StatsD` is incremented in the process of a job.

Reply:
I didn't actually find a consistent pattern for this but I refactored the notifications keys to use dots for status so the querying makes more sense in https://github.com/department-of-veterans-affairs/vets-api/pull/16359/commits/ef842e7edbd7d232772a163ae1f5711333444457

Reply:
I was thinking of something along the lines of how StatsD keys are implemented, for example, in [app/sidekiq/decision_review/form4142_submit.rb](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/sidekiq/decision_review/form4142_submit.rb). Setting a single constant and appending the appropriate outcome when StatsD is incremented would also have the benefit of being slightly more DRY.